src/nvim/ex_cmds.c

```diff
@@ -5357,31 +5345,31 @@ static void helptags_one(char_u *const dir, const char_u *const ext,
       fprintf(fd_tags, "!_TAG_FILE_ENCODING\tutf-8\t//\n");
     }
```

Comment:
In the case of "no tags", before this change, this line would still write to the  mostly-empty file. Now it doesn't. That's probably ok I guess.